qubesbuilder/plugins/fetch/scripts/verify-file

```diff
@@ -114,7 +114,7 @@ elif [ -n "${UNTRUSTED_SIGNATURE_FILE}" ] && [ -n "${PUBKEY_FILE}" ]; then
 
     # Import pubkey
     keyring="$(mktemp -u)"
-    sq keyring merge --output "$keyring" "${PUBKEY_FILE[@]}"
+    sq toolbox keyring merge --output "$keyring" "${PUBKEY_FILE[@]}"
```

Comment:
This needs to be a bit more flexible - Debian still has the old version (and likely will have for quite some time), so old syntax still need to be supported. I don't see an elegant way of checking but `sq help|grep toolbox` might be enough.